dip-ct.md

```diff
@@ -0,0 +1,307 @@
+<pre>
+  DIP: XXX
+  Title: Confidential Transactions
+  Author: Duke Leto
+  Comments-Summary: No comments yet.
+  Status: In Progress
+  Type: Consensus
+  Created: 2024-08-13
+  License: MIT License
+</pre>
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+1. [Motivation](#motivation)
+1. [Conventions](#conventions)
+1. [Prior Work](#prior-work)
+1. [Consensus Protocol Changes](#consensus-protocol-changes)
+1. [Risks](#risks) 
+1. [References](#references)
+1. [Copyright](#copyright)
+
+## Abstract
+
+We outline a Confidential Transaction scheme for Dash. After deployment and
+activation, Dash will be able to make transactions which do not leak
+the amount being transferred.
+
+## Motivation
+
+Currently Dash transactions can optionally use CoinJoin to increase the privacy of transactions but it
+leaves much to be desired. This is because CoinJoin is a mixing protocol which
```

Comment:
I would refrain from calling Coinjoin a mixing protocol. Our implementation of coinjoin does not mix transactions like a mixer would (puts all coins together mixes them and spits them out). Instead coins are self transferred repeatedly so that it's hard to trace the input of a coinjoined transaction. I would instead call it a non-custodial, multi-round CoinJoin process.